aws/aws-quiz.md

```diff
@@ -227,7 +227,7 @@
 #### Q29. A new developer has been added to the team and you have been asked to provide access to the organization's AWS account. What is the best practice for granting access?
 
 - [ ] Give the new developer the IAM login that is assigned to the development team. This IAM user should already include all of the policies that a developer would need.
-- [ ] Create a IAM user for the new developer. Manually assign policies to the new IAM user account.
+- [ ] Create a IAM user for the new developer. Manually assign policies to the new IAM user acunt.
```

Comment:
The initial word was correct, we don't need to change it. Please, fix this typo.

```suggestion
- [ ] Create a IAM user for the new developer. Manually assign policies to the new IAM user account.
```